openedx/core/djangoapps/appsembler/sites/api.py

```diff
@@ -233,9 +234,9 @@ def get(self, request):
         if user_email and organization_name:
             try:
                 organization = Organization.objects.get(name=organization_name)
-                mapping = UserOrganizationMapping.objects.get(user__email=user_email, organization=organization)
-                return Response({'username': mapping.user.username}, status=status.HTTP_200_OK)
-            except (Organization.DoesNotExist, UserOrganizationMapping.DoesNotExist):
+                user = get_organization_user_by_email(email=user_email, organization=organization)
+                return Response({'username': user.username}, status=status.HTTP_200_OK)
+            except User.DoesNotExist:
```

Comment:
`ObjectDoesNotExist` catches all models and in this case we have two models: User and Organization.

```suggestion
            except ObjectDoesNotExist:
```

Reply:
Done